relay-general/src/protocol/measurements.rs

```diff
@@ -0,0 +1,171 @@
+use regex::Regex;
+
+use crate::types::{Annotated, Error, FromValue, Object, Value};
+
+lazy_static::lazy_static! {
+    static ref MEASUREMENT_NAME: Regex = Regex::new("^[a-z0-9-._]+$").unwrap();
+}
+
+#[derive(Clone, Debug, Default, PartialEq, Empty, FromValue, ToValue, ProcessValue)]
+#[cfg_attr(feature = "jsonschema", derive(JsonSchema))]
+pub struct Measurement {
+    /// Value of observed measurement value
+    #[metastructure(required = "true", skip_serialization = "empty")]
+    pub value: Annotated<f64>,
```

Comment:
If this is required, then we should never skip serialization.

Reply:
Fixed in https://github.com/getsentry/relay/pull/724/commits/414836966b39c48e4ee32c1490b2b9aa0979b8a5